meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core

```diff
@@ -0,0 +1,21 @@
+# Open source to the core
+
+- __Speaker__: Ivo Lukač & Edi Modrić
```

Comment:
Having 2 speakers for a 25 minute slot feels too much... as Ivo has more experience talking and I feel Edi is much closer to the code part, I would suggest that Edi does it himself